helm/templates/controller/statefulset.yaml

```diff
@@ -85,8 +85,7 @@ spec:
               mountPath: "/account"
             {{- end }}
           livenessProbe:
-            httpGet:
-              path: /pinot-controller/admin
```

Comment:
so this just opens a tcpsocket for liveness? That seems fairly generic - would we be marking it as live prematurely?

Reply:
Not necessarily, the tcp socket check will make sure pinot process is up and running. The port is opened at the start but the health takes a lot of time before returning success. We have not changed the readiness probe. It will start serving only readiness check is passed. Readiness probe will sure that the next pod is restarted only when the first one is completely initialized.

https://github.com/hypertrace/pinot/issues/46

Reply:
The part about readiness preventing premature serving I'm certainly in agreement with, but I'm just trying to answer my own questions on how this works. Would opening that socket fail in all of the following scenarios - the JVM crashed, the JVM deadlocked, or the port was misconfigured and the liveness probe is querying the wrong port? Assuming the answer is yes which I think it is, I'm good with this. If it's no, it's better than what we have right now but I'd want us to leave the issue open.